spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :user do
+    email { 'user@email.com' }
```

Comment:
Use sequences to prevent your test users from having the same email address. (You'll hit this problem as soon as you generate more than one user in a test.)
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences